website/docs/r/linux_virtual_machine_scale_set.html.markdown

```diff
@@ -232,7 +242,9 @@ The following arguments are supported:
 
 A `additional_capabilities` block supports the following:
 
-* `ultra_ssd_enabled` - (Optional) Should the capacity to enable Data Disks of the `UltraSSD_LRS` storage account type be supported on this Virtual Machine Scale Set? Defaults to `false`. Changing this forces a new resource to be created.
+* `hibernation_enabled` - (Optional) Should the VM be allowed to enter hibernation mode? Possible values are `true` or `false`. Defaults to `false`. Changing this forces a new resource to be created.
```

Comment:
should this be
```suggestion
* `hibernation_allowed` - (Optional) Should the VM be allowed to enter hibernation mode? Possible values are `true` or `false`. Defaults to `false`. Changing this forces a new resource to be created.
```

as enabled implies "hibernate"

Reply:
I think that `enabled` is the correct verb here, I believe everyone knows what hibernation mode is and can understand what this is doing field is doing. I could be wrong, and I will change it if you really want me too, but I personally think this is fine the way it is.